src/index.js

```diff
@@ -0,0 +1,181 @@
+const tempValue = document.getElementById('tempValue');
+const landscape = document.getElementById('landscape');
+const currentTempButton = document.getElementById('currentTempButton');
+const cityNameInput = document.getElementById('cityNameInput');
+const headerCityName = document.getElementById('headerCityName');
+const skySelect = document.getElementById('skySelect');
+const sky = document.getElementById('sky')
+const gardenSection = document.getElementById('gardenSection');
+const cityNameReset = document.getElementById('cityNameReset');
+
+
+const defaultTemp = () => {
+  const defaultTemperature = 70;
+  tempValue.textContent = defaultTemperature;
+  updateTempColor(defaultTemperature);
+  updateLandscape();
+}
+
+// WAVE 2
+const updateTemp = (increment) => {
+  let currentTemp = parseInt(tempValue.textContent);
+  let newTemp = currentTemp + increment;
+
+  tempValue.textContent = newTemp;
+
+  updateTempColor(newTemp);
+}
+
+const updateTempColor = (temperature) => {
+  tempValue.classList.remove('red','orange','yellow','green','turqoise');
+
+  if (temperature > 79) {
+    tempValue.classList.add('red');
+  } else if (temperature > 69) {
+    tempValue.classList.add('orange');
+  } else if (temperature > 59) {
+    tempValue.classList.add('yellow');
+  } else if (temperature > 49) {
+    tempValue.classList.add('green');
+  } else {
+    tempValue.classList.add('turqoise');
+  }
+
+  updateLandscape();
+}
+
+const updateLandscape = () => {
+  let currentTemp = parseInt(tempValue.textContent);
+
+  if (currentTemp >= 79) {
+    landscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (currentTemp >= 69) {
+    landscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (currentTemp >= 59) {
+    landscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else {
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+}
+
+// WAVE 3
+const updateCityName = () => {
+  cityNameInput.addEventListener("input", updateValue);
+
+  function updateValue(e) {
+    headerCityName.textContent = e.target.value;
+  }
+}
+
+// WAVE 4
+const findLatitudeAndLongitude = async (query) => {
+  try {
+    const response = await axios
+      .get('http://127.0.0.1:5000/location', {
+        params: {
+          q: query,
+          format: 'json'
+        }
+      });
+
+    const latitude = response.data[0].lat;
+    const longitude = response.data[0].lon;
+    console.log('success in findLatitudeAndLongitude!', 'latitude:', latitude, 'longitude:', longitude);
+
+    return await findWeather(latitude, longitude);
+  } catch (error) {
+    console.log(error);
+    return null;
+  }
+}
+
+const findWeather = async (lat, lon) => {
+  try {
+    const response = await axios
+      .get('http://127.0.0.1:5000/weather', {
+        params: {
+          format: 'json',
+          lat: lat,
+          lon: lon,
+        }
+      });
+
+    console.log('success in findWeather!', response.data);
+    return response.data.main.temp;
```

Comment:
By the OpenWeather documentation, the `temp` value is being returned in Kelvin units and should be converted to Fahrenheit if we want it to work with the temperature changes of the weather garden we created.